modules/data.land/R/cohort2pool.R

```diff
@@ -0,0 +1,55 @@
+##' @name cohort2pool
+##' @title cohort2pool
+##' @export
+##' @param veg_file standard cohort veg_file
+##' @param allom_param parameters for allometric equation, a and b
+##' @author Saloni Shah
+
+cohort2pool <- function(veg_file,allom_param =NULL) {
+  
+  dat <- readRDS(veg_file)
+  
+  #Building Site ID from past directories
+  
+  path <- dirname(veg_file)
+  last_dir <- basename(path)
+  nums_id <- strsplit(last_dir,"[^[:digit:]]")
+  base_id <- nums_id[[1]][length(nums_id[[1]])]
+  suffix <- nums_id[[1]][(length(nums_id[[1]])-1)]
+  if (suffix == 1) {
+    siteid <- as.numeric(paste0(suffix, "0000", base_id))
+  } else if (siteid ==0) {
+    siteid <- as.numeric(base_id)
+  }
```

Comment:
This if/else doesn't make sense to me. 

First, why are you only building the site_id if the server is BU (this code needs to work for all servers, not just 1). There should never be server-specific hacks like this in the code

Second, why are you assuming a fixed number of digits between the suffix and base ID?  siteid should just be
```
as.numeric(suffix)*10^9 + as.numeric(base_id)
```
and that should work pretty consistently across servers

Third, the ```else if``` cannot ever be true because it's based on the siteid but the siteid is not yet defined unless the previous if was true. I think you meant suffix == 0, but as noted previously you don't need to handle this as a separate case